architecture/introduction-openshift-architecture.adoc

```diff
@@ -0,0 +1,26 @@
+[id="architecture"]
+= {product-title} architecture
+include::modules/common-attributes.adoc[]
+:context: architecture-intro
+toc::[]
+
+include::modules/openshift-introduction.adoc[leveloffset=+1]
+include::modules/container-application-benefits.adoc[leveloffset=+2]
+include::modules/kubernetes-introduction.adoc[leveloffset=+2]
+include::modules/openshift-benefits.adoc[leveloffset=+2]
+include::modules/understanding-openshift-installation.adoc[leveloffset=+1]
+include::modules/running-simple-openshift-installation.adoc[leveloffset=+2]
+include::modules/running-modified-openshift-installation.adoc[leveloffset=+2]
+// Standalone?
+include::modules/following-openshift-installation.adoc[leveloffset=+2]
+include::modules/completing-openshift-installation.adoc[leveloffset=+2]
+include::modules/understanding-openshift-control-plane.adoc[leveloffset=+1]
+include::modules/understanding-node-roles.adoc[leveloffset=+2]
+include::modules/understanding-workers-masters.adoc[leveloffset=+2]
+include::modules/defining-workers.adoc[leveloffset=+3]
+include::modules/defining-masters.adoc[leveloffset=+3]
```

Comment:
I'd strip the procedural information and merge the concepts in lines 17 through 21 - the content makes sense together and probably would not be reused separately.